### ISSUE TITLE
Indala 224 card not being read

### ISSUE THREAD

**Issue by Divide-By-0** [flipperdevices/flipperzero-firmware#2386]
### Describe the bug.

I have a dual Indala 125 kHz ID and Mifare Classic 13.56 mHz card, and the Mifare classic reads fine on NFC. The Indala should read on the Extra Actions menu but nothing is detected.

### Reproduction

Switch to Indala mode
Try to read
It doesn't read

### Target

Indala card, len 224. Proxmark 3 recognizes it as chipset detection: EM4x05 / EM4x69.
The proxmark sometimes confueses it for a `COTAG Found: FC 240, CN: 64702 Raw: EBE3...`

### Logs

There's nothing shown on the screen, what command is most useful for me to put logs?

### Anything else?

I can read it on the Proxmark 3 (but not write it), it starts with 80000003...

**Comment by ghost** [comment]
No entiendo

**Comment by DrZlo13** [comment]
Yes, Indala-224 is not supported by the LF-RFID protocol library. We need to wait for someone to implement it.

**Comment by Divide-By-0** [comment]
I see, is it still possible to emulate/write if I know the ground truth code from being previously read?

**Comment by Codeplaya** [comment]
Looking at the github, I understand that Flipper is trying to read only Indala cards with PSK1 modulation.

At the same time, looking at [Proxmark 3 commands code](https://github.com/Proxmark/proxmark3/blob/555fa197730c061bbf0ab01334e99bc47fb3dc06/client/cmdlfindala.c), it lists 4 different modulations for 4 different types of Indala cards right in the beginning of the code: “PSK1, rf/32, 64 or 224 bits (known)”

In their [demodulation code](https://github.com/Proxmark/proxmark3/blob/555fa197730c061bbf0ab01334e99bc47fb3dc06/common/lfdemod.c) on line 1779 there is demodulation code for Indala 64-bit and 224-bit cards. There is also a comment by Marshmallow that 224 formats are typically PSK2.

Maybe it is helpful for implementing Indala higher-bit cards in Flipper.
